xml/System.Threading.Tasks.Sources/IValueTaskSource.xml

```diff
@@ -114,11 +114,11 @@
         <Parameter Name="flags" Type="System.Threading.Tasks.Sources.ValueTaskSourceOnCompletedFlags" />
       </Parameters>
       <Docs>
-        <param name="continuation">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="token">To be added.</param>
-        <param name="flags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="continuation">The continuation to invoke when the operation has completed.</param>
+        <param name="state">The state object to pass to &lt;paramref name="continuation" /> when it's invoked.</param>
```

Comment:
Is this &lt; correct when the paramref then ends with /> ?

Reply:
No, it's not correct, @stephentoub. I've changed it.